src/Commands/CreateUserCommand.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Commands;
+
+use App\Exceptions\User\UnableToCreateUserException;
+use App\Exceptions\ValueObject\Username\InvalidUsernameException;
+use App\Repository\User\UserManagerInterface;
+use App\ValueObject\Username;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+final class CreateUserCommand extends Command
+{
+    private $userManager;
+
+    public function __construct(UserManagerInterface $userManager)
+    {
+        parent::__construct();
+
+        $this->userManager = $userManager;
+    }
+
+    public function configure(): void
+    {
+        $usernameHelp = Username::getCreationConstraint();
+        $this
+            ->setName('app.users.create')
```

Comment:
Why not namespacing command names like this: `app:users:create` ?
It is the Symfony way, also it is displayed better in help page(when doing `bin/console`), all our commands will be grouped in the `app` namespace.